### DISCUSSION TITLE
Import-Module : The required module 'dbatools.library' is not loaded.

### DISCUSSION THREAD

**Discussion by thuraisiva** [dataplat/dbatools#8722] [General] [answered]
I have downloaded dbatools from GitHub. Unziped , Renamed as dbatools. Copy to module folder. then ran get-childitem with unbloked folder. but still getting following error
Import-Module : The required module 'dbatools.library' is not loaded. Load the module or remove the module from
'RequiredModules' in the file 'C:\Program Files\WindowsPowerShell\Modules\dbatools\dbatools.psd1'.
At line:1 char:1
+ Import-Module dbatools
+ ~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : ResourceUnavailable: (C:\Program File...s\dbatools.psd1:String) [Import-Module], Missing
   MemberException
    + FullyQualifiedErrorId : Modules_InvalidManifest,Microsoft.PowerShell.Commands.ImportModuleCommand

**Comment by potatoqualitee** [comment, marked as answer]
we're knee deep working on dbatools 2.0 is not out yet so this is expected behavior. I'd recommend downloading the nuget file from https://dbatools.io/gallery, rename it to .zip, extract it and go about the same directions. a prerelease is coming out pretty soon and then youll just be able to use install-module

**Comment by mfread** [comment]
Thanks potato. This solution worked for me. Was ripping my hair out for a few hours researching why (as it used to work with an older version). I'm deploying onto a client's jumpbox so can only copy up a zip and do a manual install.
